app/components/Lock.js

```diff
@@ -0,0 +1,26 @@
+var React = require('react');
+var classnames = require('classnames');
+
+var Lock = React.createClass({
+    render: function() {
+        const classes = classnames({
+            'tilt-icon': true,
+            'tilt-lock': true,
+            [this.props.className]: !!this.props.className
+        });
```

Comment:
you can pass the this.props.className as a second argument to the classnames function and it will get added if it's present.

classnames({ ......... }, this.props.className);